scripts/preconversion_assessment_script.py

```diff
@@ -226,29 +373,28 @@ def main():
         with open(C2R_REPORT_FILE, "r") as handler:
             data = json.load(handler)
 
-        output.report_json = data
-
         print("Collecting status.")
-        combined_levels = collect_report_level(action_results=data["action_results"])
-        output.status = combined_levels[0]
+        combined_levels = collect_report_level(action_results=data["actions"])
+
         # Set the first position of the list as being the final status, that's
         # needed because `collect_report_level` will sort out the list with the
         # highest priority first.
-
+        output.status = combined_levels[0]
```

Comment:
thinking if the function shouldn't return the highest one already, do we need the rest? if the report_level = the highest one, then I would just return the level